js/huobi.js

```diff
@@ -914,43 +914,132 @@ module.exports = class huobi extends Exchange {
                     'BTC': 'BTC',
                     'USDT': 'TRC20',
                 },
+                'hasUniqueNetworkIds': true,
                 'networks': {
-                    // by displaynames
-                    'ALGO': 'ALGO',
-                    'ALGORAND': 'ALGO',
+                    'ACALA': 'ACA',
+                    'ALGORAND': 'ALGO', // 'ALGO'
```

Comment:
in such cases, where i have `// 'XYZ'` placed aside these lines, it means that name is also somewhat popular and we could add that in a new line, like :
```
'ALGORAND': 'ALGO',
'ALGO': 'ALGO',
```
but on the other hand, I think it's better to have one unified-network-code (wherever possible) so users shouldn't have burden to learn bunch of "alternative unified network codes" and dont offer then `ALGO, ALG, ALGORAND` as an aliases. no, I think it's better we had the one official name `ALGORAND` as unified network code and thats.

i know we can't do that for BTC,ETH,TRON (as they are exclusions now), but I mean for the rest of secondary networks, except those exclusions.